src/test/java/se/sundsvall/billingdatacollector/integration/opene/OpenEIntegrationTest.java

```diff
@@ -55,15 +55,15 @@ void getErrandIds(@Load("/open-e/flow-instances.xml") final String xml) {
 	@Test
 	void getErrand(@Load("/open-e/flow-instance.internal.xml") final String xml) {
 		when(mockOpenEClient.getErrand("123456")).thenReturn(xml.getBytes(UTF_8));
-		when(mockMapper.mapToBillingRecord(any())).thenReturn(BillingRecordWrapper.builder().build());
+		when(mockMapper.mapToBillingRecordWrapper(any())).thenReturn(BillingRecordWrapper.builder().build());
```

Comment:
`any(byte[].class)` ?

Reply:
Just a rename of the method but you are correct